freetextresponse/freetextresponse.py

```diff
@@ -322,13 +362,12 @@ def _get_indicator_class(self):
         """
         Returns the class of the correctness indicator element
         """
-        result = ''
-        if self.count_attempts == 0:
-            result = 'unanswered'
-        elif self._determine_credit() == Credit.zero:
-            result = 'incorrect'
-        else:
-            result = 'correct'
+        result = 'unanswered'
+        if self.display_correctness and self._word_count_valid():
```

Comment:
The indicator_class variable set by that function only impacts client side refresh and submit.   It is currently in the server side save function however and doesn't need to be.  I'll remove that.